<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changelesson</class>
 <widget class="QWidget" name="changelesson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改选课</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>171</width>
     <height>250</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_17">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_63">
      <item>
       <widget class="QLabel" name="label_63">
        <property name="text">
         <string>学生学号：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_63">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_63"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_43">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_57">
      <item>
       <widget class="QLabel" name="label_57">
        <property name="text">
         <string>课程号：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_57">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_57"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_41">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_58">
      <item>
       <widget class="QLabel" name="label_58">
        <property name="text">
         <string>课程名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_58">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_58"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_42">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_59">
      <item>
       <widget class="QLabel" name="label_59">
        <property name="text">
         <string>学分：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_59">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_59"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>160</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_18">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_60">
      <item>
       <widget class="QLabel" name="label_60">
        <property name="text">
         <string>学期：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_60">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_60"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_61">
      <item>
       <widget class="QLabel" name="label_61">
        <property name="text">
         <string>教室：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_61">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_61"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_44">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_62">
      <item>
       <widget class="QLabel" name="label_62">
        <property name="text">
         <string>成绩：</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_62">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_62"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
